.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
 - pip install datalad
 - pip freeze
 - datalad install -r .
+- pip install -r scripts/dats_validator/requirements.txt
```

Comment:
Putting this line above `pip freeze` will alllow `pip freeze` to list the installed packages from pip